src/routes/genres.routes.ts

```diff
@@ -143,14 +159,20 @@ router.put('/:id', (req: Request, res: Response, next: NextFunction) => {
  *       500:
  *         description: Internal Server Error
  */
-router.delete('/:id', (req: Request, res: Response, next: NextFunction) => {
-  const genreId = req.params.id;
+router.delete(
+  '/:id',
+  async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const genreId = req.params.id;
+
+      const deletedGenre: Genre | null =
+        await GenreModel.findByIdAndDelete(genreId);
```

Comment:
If deletedGenre will be null?
You need to add this checking and return appropriate status

Reply:
added check for null